.github/workflows/ci.yml

```diff
@@ -69,11 +80,43 @@ jobs:
       - run: git diff
       - run: git status --porcelain
       - run: test -z "$(git status --porcelain)"
-on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-  push:
-    branches:
-      - main
+  semantic-release:
+    name: Semantic Release
+    runs-on: ubuntu-22.04
+    # Avoid semantic release for forked repository
+    if: github.event_name != 'pull_request' && github.repository == 'serious-scaffold/ss-python'
+    permissions:
+      contents: write
+      id-token: write
+      issues: write
+      pull-requests: write
+    needs: ci
```

Comment:
```suggestion
    needs: [ci, consistency]
```

Reply:
release job needs consistency for ss-python.

Reply:
I changed to multiline list, thanks for pointing out.